datasets/_global/tokyo_mou_detention/crawler.py

```diff
@@ -0,0 +1,94 @@
+from datetime import datetime
+
+from zavod import Context, helpers as h
+
+YEARS = [2024, 2025]
+MONTHS = range(1, 13)
+
+
+def is_future_month(year: int, month: int, now: datetime) -> bool:
+    return (year > now.year) or (year == now.year and month > now.month)
+
+
+def emit_linked_org(context, vessel_id, names, role):
+    for name in h.multi_split(names, ";"):
+        org = context.make("Organization")
+        org.id = context.make_id("org", name)
+        org.add("name", name)
+        context.emit(org)
+
+        link = context.make("UnknownLink")
+        link.id = context.make_id(vessel_id, org.id, role)
+        link.add("role", role)
+        link.add("subject", org.id)
+        link.add("object", vessel_id)
+        context.emit(link)
+
+
+def crawl_row(context: Context, clean_row: dict):
+    ship_name = clean_row.pop("ship_name")
+    imo = clean_row.pop("imo_no")
+    company_name = clean_row.pop("company")
+    ros = clean_row.pop("related_ros")
+    cs = clean_row.pop("classification_society")
```

Comment:
try and name variables so that someone reading it can have a fairly good guess what it refers to. I need a reference with a better author, but this works for now https://gist.github.com/klaaspieter/12cd68f54bb71a3940eae5cdd4ea1764

```suggestion
    related_ros = clean_row.pop("related_ros")
    class_soc = clean_row.pop("classification_society")
```